fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/Select.java

```diff
@@ -10,16 +10,25 @@
 import static com.ibm.fhir.database.utils.query.SqlConstants.SELECT;
 import static com.ibm.fhir.database.utils.query.SqlConstants.SPACE;
 
-import com.ibm.fhir.database.utils.query.expression.Predicate;
-import com.ibm.fhir.database.utils.query.expression.PredicateExpression;
+import com.ibm.fhir.database.utils.api.IDatabaseTranslator;
+import com.ibm.fhir.database.utils.derby.DerbyTranslator;
+import com.ibm.fhir.database.utils.query.expression.StatementRenderer;
+import com.ibm.fhir.database.utils.query.expression.StringStatementRenderer;
+import com.ibm.fhir.database.utils.query.node.ExpNode;
+import com.ibm.fhir.database.utils.query.node.ExpNodeVisitor;
 
 /**
  * Representation of a select statement built by {@link SelectAdapter#build()}
  */
 public class Select {
+    // Need a translator to render statement for debugging
+    private static final IDatabaseTranslator TRANSLATOR = new DerbyTranslator();
+
     // The fields, expressions we are selecting
     private final SelectList selectList = new SelectList();
 
+    private boolean distinct;
```

Comment:
Distinct does... 

and it should probably have a default of false, or have the autobox version Boolean

Reply:
boolean members will be false by default. But I made it final and assign in each constructor now.

Reply:
sounds good...